test/functions/sungeun/procEquals_c_string.good

```diff
@@ -0,0 +1 @@
+procEquals_c_string.chpl:1: error: Can't overload assignments for pointer types
```

Comment:
I'm not understanding what in this PR caused this change either.

Reply:
See - https://github.com/chapel-lang/chapel/pull/19306#issuecomment-1081054605

> I've been trying to improve the `Can't overload assignments for class types` error but haven't quite stuck the landing. Anyway this is a side issue for the design proposal here.

Like I said in https://github.com/chapel-lang/chapel/pull/19306#discussion_r845269625, I am expecting to remove that effort from this PR. However I am not expecting it to impact the evaluation of the language design change.